pdi-dataservice-client-api/pom.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>pentaho</groupId>
+    <artifactId>pdi-dataservice-client</artifactId>
+    <version>8.1.0.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>pdi-dataservice-client-plugin-api</artifactId>
+  <version>8.1.0.0-SNAPSHOT</version>
+  <packaging>bundle</packaging>
+  <name>Pentaho Community Edition Project: ${project.artifactId}</name>
+  <description>API for the PDI Data Services</description>
+  <url>http://www.pentaho.com</url>
+  <licenses>
+    <license>
+      <name>Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+      <comments>A business-friendly OSS license</comments>
+    </license>
+  </licenses>
+  <dependencies>
+    <dependency>
+      <groupId>pentaho</groupId>
+      <artifactId>metastore</artifactId>
+      <version>${pdi.version}</version>
```

Comment:
Please use the proper version property for the metastore artifact. You can check that here: https://github.com/pentaho/build-resources/blob/8.1.0.0/version.properties#L94

Reply:
That was changed in the core pom.xml in this commit. It was an eng-ops task, so somehow I assume that it's better this way, regarding versions management.
https://github.com/pentaho/pdi-dataservice-plugin/commit/81080e9c2f96790d2ef86aad4c017377a86103a2

Reply:
Before that commit it was already using the property version for kettle instead of the metastore so that can be an error. Perhaps a find and replace where we didn't notice the artifact was for the metastore? Can you please confirm which property to use with @pentaho/2-1b / @smaring?

Reply:
it was wrong in the first place, please change to `pentaho-metastore.version`

Reply:
Will do... I will also change it in the pdi-dataservice-client/pom.xml